packages/doc/docs/api-plugin/pagination.mdx

```diff
@@ -0,0 +1,27 @@
+# Pagination
+## Why use pagination?
+
+In some cases, we execute SQL queries to list all entities, which may result in a large amount of data. Instead of sending all the data to our clients, we can paginate the results and send only the necessary portion. This approach is a common design in modern API servers.
+
+## Implement Manually
+
+You can implement your own pagination strategies by modifying your SQL queries. For example, you can use parameters to implement a simple offset-based pagination:
+
+```sql
+SELECT * FROM users
+-- highlight-next-line
+LIMIT {{ context.params.limit }}
+-- highlight-next-line
+OFFSET {{ context.params.offset }}
```

Comment:
```suggestion
SELECT * FROM users
// highlight-next-line
LIMIT {{ context.params.limit }}
// highlight-next-line
OFFSET {{ context.params.offset }}
```

Reply:
addressed in e265cf5eb3ab7f6230a5e4c419181f346638a6e4